types/sql.js/sql.js-tests.ts

```diff
@@ -1,26 +1,28 @@
 import fs = require('fs');
-import initSqlJs from 'sql.js';
 
-var DB_PATH = 'data.db';
+import * as initSqlJs1 from 'sql.js';
+import initSqlJs2 from 'sql.js';
+import initSqlJs3 = require('sql.js');
+
+const DB_PATH = 'data.db';
 
 initSqlJs().then(SQL => {
```

Comment:
Sorry, I don't quite understand. I already imported `initSqlJs` in line 3, as suggested by you.

Reply:
It's only a comment. What I meant was:
1. If `initSqlJs` is not imported, here this identifier has the semantic of accessing a global object. It is injected to the global scope only when it is *not* loaded as a module (e.g. using `<script>`). So compilation passes, but it will throw a runtime error when you compile the test code and run in a Node environment (which is quite popular for tests).
2. If the symbol is correctly imported, for example, being fixed as suggested, then there's nothing to worry about. ← Now we are here

Reply:
Alright, thanks! :slightly_smiling_face: